src/components/dropdown/dropdown.js

```diff
@@ -288,11 +288,9 @@ const Dropdown = (props) => {
   const _handleClose = () => {
     setOpen(false);
 
-    this.setState({ open: false }, () => {
-      if (typeof onClosed === 'function') {
-        onClosed();
-      }
-    });
+    if (typeof onClosed === 'function') {
+      onClosed();
+    }
```

Comment:
This is a callback on setting the state that needs to be handled in a useEffect to ensure this is only executed after the state change (i.e the `setOpen` state change here). 

Regardless, on checking the code, this is actually already set up in the useEffects above, hence can be deleted.